Libraries/Opc.Ua.PubSub/Opc.Ua.PubSub.csproj

```diff
@@ -36,6 +36,7 @@
   </ItemGroup>
   
   <ItemGroup>
+    <PackageReference Include="MQTTnet" Version="3.0.14" />
```

Comment:
would be nice if the dependency to the MQTT lib could be factored out, otherwise its now tied to this one. Some application may jut want to use the encoding/decoding but handle the transport self. (future)

Reply:
right. good idea for upcoming inprovements